playground-common/src/Control/Monad/Web.hs

```diff
@@ -22,18 +25,25 @@ class Monad m =>
   doRequest :: Manager -> Request -> m (Either Text (Response LBS.ByteString))
 
 instance MonadWeb IO where
-  makeManager = newManager $ tlsManagerSettings {managerModifyRequest = pure . addUserAgent}
+  makeManager = newManager $ tlsManagerSettings
+    { managerModifyRequest = pure . addUserAgent
+    , managerResponseTimeout = maybe
```

Comment:
There's a lot of such duplicated code. However didn't notice that the project actively uses lenses, so decided to keep things simple at the moment.